### ISSUE TITLE
Add indentation support of tables inside lists

### ISSUE THREAD

**Issue by darkriszty** [darkriszty/MarkdownTablePrettify-VSCodeExt#47]
When tables are inside lists they lose their indentation after formatting. A left pad / indentation has to be supported when the table has at least a left border in the output (`|`).

Expected behavior for mixed indentation scenarios:
* When at least half the rows have indentation, use indentation in the output.
* When different indentation styles are used, use the one that is most frequent in the output.
---

* Sample input
    ```
    * List item
        |Type | Range| Size|
        |-|-|-|
        |sbyte | -128 to 127| Signed 8-bit integer|
        |byte| 0 to 255| Unsigned 8-bit integer|
        |char| U+0000 to U+ffff|	Unicode 16-bit character|
    ```
* Current output
    ```
    * List item
    | Type  | Range            | Size                     |
    |-------|------------------|--------------------------|
    | sbyte | -128 to 127      | Signed 8-bit integer     |
    | byte  | 0 to 255         | Unsigned 8-bit integer   |
    | char  | U+0000 to U+ffff | Unicode 16-bit character |
    ```
* Expected output
    ```
    * List item
        | Type  | Range            | Size                     |
        |-------|------------------|--------------------------|
        | sbyte | -128 to 127      | Signed 8-bit integer     |
        | byte  | 0 to 255         | Unsigned 8-bit integer   |
        | char  | U+0000 to U+ffff | Unicode 16-bit character |
    ```

**Comment by darkriszty** [comment]
Included in v3.2.0

**Comment by magnetikonline** [comment]
Thanks for this @darkriszty 👍
